integration_tests/e2e/refer.cy.ts

```diff
@@ -349,6 +350,92 @@ context('Refer', () => {
     })
   })
 
+  describe('When adding programme history', () => {
+    describe('and selecting a programme', () => {
+      const courses = courseFactory.buildList(4)
+      const prisoner = prisonerFactory.build({
+        firstName: 'Del',
+        lastName: 'Hatton',
+      })
+      const person = personFactory.build({
+        currentPrison: prisoner.prisonName,
+        name: 'Del Hatton',
+        prisonNumber: prisoner.prisonerNumber,
+      })
+      const referral = referralFactory.started().build({ prisonNumber: person.prisonNumber })
+      const path = referPaths.programmeHistory.new({ referralId: referral.id })
+
+      beforeEach(() => {
+        cy.task('stubCourses', courses)
+        cy.task('stubPrisoner', prisoner)
+        cy.task('stubReferral', referral)
+
+        cy.signIn()
+
+        cy.visit(path)
+      })
+
+      it('shows the select programme form page', () => {
+        const selectProgrammePage = Page.verifyOnPage(SelectProgrammePage, { courses })
+        selectProgrammePage.shouldHavePersonDetails(person)
+        selectProgrammePage.shouldContainNavigation(path)
+        selectProgrammePage.shouldContainBackLink('#')
```

Comment:
Now that we have the programme history index page, shall we put these back links in? Happy to do that in a separate PR if you'd rather but shouldn't be a big change.

Reply:
Aye good shout